articles/network-types-topologies/index.md

```diff
@@ -0,0 +1,91 @@
+Due to this reason, cables are used as a transmission medium. Ethernet cables are supported by many devices that have the same interface. They are also the most commonly used cables for connecting modems, routers, etc. Their signal strength is consistent and the communication network is also stable. Thus making them a practical choice for using wired LAN.
+
+![Example of LAN](/engineering-education/network-types-topologies/lan.jpg)
+
+#### Wireless Local Area Network (WLAN)
+[WLAN(Wireless LAN)](https://en.wikipedia.org/wiki/Wireless_LAN) works the same way as the LAN. The network spread is across a limited area. But this type of network does not require any cables or optic fibers. Unlike the LAN, here the mode of transmission is wireless. For wireless transmission, [Wi-Fi(Wireless Fidelity)](https://en.wikipedia.org/wiki/Wi-Fi) is used. Workstations can be connected to the internet with the help of Wi-Fi Routers and Adapters. Devices that support wireless transmission can also share data from their memory drives. Some of the devices that support wireless transmission are smartphones, smart-watches, speakers, laptops etc.
+
+![Example of WLAN](/engineering-education/network-types-topologies/wlan.jpg)
+
+#### Metropolitan Area Network (MAN)
+The point of having [MANs](https://en.wikipedia.org/wiki/Metropolitan_area_network) is similar to LANs. One of the differences is the geographical spread of the network range. As mentioned above, LANs are confined to small areas. But, as the name suggests, MAN’s are spread across an entire metropolitan city. They can be used to share data from one point of the city to another. Business establishments, telephone companies, cable TV’s use this type of networks.  
+
+![Example of MAN](/engineering-education/network-types-topologies/man.jpg)
+
+#### Wide Area Network (WAN)
+This type of network is currently the one mostly used. [WAN](https://en.wikipedia.org/wiki/Wide_area_network) is nothing but a collection of several LANs linked together. These networks are spread across countries and other parts of the world. Some of them are connected through optic fibers while some are connected through satellites. WANs are used by government agencies, multinational companies for exchanging information overseas. Since the network is spread across the world, it is maintained by many firms and companies. When the network is this big, it is prone to failures and system shutdowns once in a while. The biggest WAN in existence is the Internet.   
+
+![Example of WAN](/engineering-education/network-types-topologies/wan.jpg)
+
+ Configuring a computer network can also depend on the extent of the network, the distance between devices and the number of devices used. While LANs, WLANs, and WANs are most commonly used, there are some [other](https://www.belden.com/blog/smart-building/network-types) computer networks that can also be used based on our networking parameters.

```

Comment:
This sentence seems like a transition - but Im not quite sure - is the sentence suggesting that the topologies detailed below are the others that were mentioned? 
If not - a short list of examples of what others? you meant with links would be helpful for any interested reader

Reply:
Well, a topology is like a design/model in which network types can be structured. The details mentioned above have nothing to do with the topologies. The above section is like a conclusion to the "different types of networks".

Yes - by "others"(which is a reference link) I meant the other network types like Personal Area Network(PAN- used by an indiviual) or Campus Area Network(CAN- used by a campus) or even a Virtual Private Network(VPN- used to create a virtual p2p connection to provide privacy to the users.)

Is the single link enough or shall I mention a couple of them(by giving separate links) and then the "others(reference link)"?